appium/lib/screen-object/accessors/element.rb

```diff
@@ -194,6 +204,10 @@ def get_element_by_text(value)
         end
       end
 
+      def get_element_children(identifier, name)
```

Comment:
Let's call this method `get_children` as we already know it can only be called for an element. Also, we can make the  argument as hash i.e. `get_children(**locator)` and then find child elements by `element.find_elements(locator.first)`

Method calls would look like: `get_children(id: 'test')` or `get_children(xpath: '\\hsgxwgd')`

Reply:
Let's also add methods for `get_parent` and `get_sibling` now that we are at it.

Reply:
Add YARD doc comments too